Sources/MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -395,6 +408,40 @@ open class CarPlayNavigationViewController: UIViewController {
         }
     }
     
+    public override func viewSafeAreaInsetsDidChange() {
+        super.viewSafeAreaInsetsDidChange()
+        
+        // Trigger update of view constraints to correctly position views like `SpeedLimitView` and
+        // `CarPlayCompassView`.
+        view.setNeedsUpdateConstraints()
+    }
+    
+    public override func updateViewConstraints() {
+        // Since there is no ability to detect current driving side mode of the CarPlay head-unit,
```

Comment:
Does `UIApplication.shared.userInterfaceLayoutDirection` reflect the driving side? CarPlay ostensibly implements the driving side adaptation feature using RTL language support.

Reply:
No matter what driving side I choose in CarPlay head-unit options I always get `.leftToRight` in `UIUserInterfaceLayoutDirection`. It's interesting that `Sony XAV-AX1000` doesn't even provide changing driving side. With `Intellidash Pro` it's possible, so I suspect that head-unit support this option only by drawing certain menus on the right.

Reply:
(For posterity, it does not.)